source/data-formats/time-series.txt

```diff
@@ -0,0 +1,174 @@
+
+- ``timeField`` is set to ``"timestamp"``
+- ``metaField`` is set to ``"location"``
+- ``granularity`` is set to ``"minutes"``
+
+.. literalinclude:: /includes/data-formats/time-series.cpp
+   :start-after: start-create-ts
+   :end-before: end-create-ts
+   :language: php
+   :dedent:
+
+To verify that you successfully created the time series collection, run
+the ``list_collections()`` method on the database and print the results:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/data-formats/time-series.cpp
+      :start-after: start-list-colls
+      :end-before: end-list-colls
+      :language: php
+      :dedent:
+
+   .. output::
+      :language: console
+      :visible: false
+
+      { "name" : "sept2023", "type" : "timeseries", "options" : { "timeseries" :
+      { "timeField" : "timestamp", "metaField" : "location", "granularity" :
+      "minutes", "bucketMaxSpanSeconds" : 86400 } }, "info" : ... }
+
+.. _cpp-time-series-store:
+
+Store Time Series Data
+----------------------
+
+You can insert data into a time series collection by using the ``insert_one()``
+or ``insert_many()`` methods and specifying the measurement, timestamp, and metadata
+in each inserted document.
+
+.. tip::
+
+   To learn more about inserting documents into a collection, see the :ref:`cpp-write-insert`
+   guide.
+
+Example
+~~~~~~~
+
+This example inserts New York City precipitation data into the ``sept2023``
+time series collection created in the :ref:`Create a Time Series Collection example
+<cpp-time-series-create-example>`. Each document contains the following fields:
+
+- ``precipitation_mm``, which stores precipitation measurements
+- ``location``, which stores location metadata
+- ``timestamp``, which stores the time of the measurement collection
+
+.. literalinclude:: /includes/data-formats/time-series.cpp
+   :start-after: start-insert-ts
+   :end-before: end-insert-ts
+   :language: php
+   :dedent:
+
+.. _cpp-time-series-query:
+
+Query Time Series Data
+----------------------
+
+You can use the same syntax and conventions to query data stored in a time 
+series collection as you use when performing read or aggregation operations on
+other collections. To find more information about these operations, see
+the :ref:`Additional Information <cpp-time-series-addtl-info>` section.
+
+.. _cpp-time-series-addtl-info:
+
+Additional Information
+----------------------
+
+To learn more about the concepts mentioned in this guide, see the
+following Server manual entries:
+
+- :manual:`Time Series </core/timeseries-collections/>`
+- :manual:`Create and Query a Time Series Collection </core/timeseries/timeseries-procedures/>`
+- :manual:`Set Granularity for Time Series Data </core/timeseries/timeseries-granularity/>`
+
+To learn more about performing read operations, see :ref:`cpp-read`.
+
+To learn more about performing aggregation operations, see the :ref:`cpp-aggregation`
+guide.
```

Comment:
Should this be commented out as a TODO or it the aggregation page ready to go and this just needs to be merged?

Reply:
It's ready! just need to finish the review process for it, but I'll merge aggregation before this PR